plugins/slick.resizer.js

```diff
@@ -119,14 +122,17 @@
         // -- 2nd bind a trigger on the Window DOM element, so that it happens also when resizing after first load
         // -- bind auto-resize to Window object only if it exist
         $(window).on('resize.grid.' + _gridUid, function (event) {
-          _self.onGridBeforeResize.notify({ grid: _grid }, event, _self);
-
-          // unless the resizer is paused, let's go and resize the grid
-          if (!_resizePaused) {
-            // for some yet unknown reason, calling the resize twice removes any stuttering/flickering
-            // when changing the height and makes it much smoother experience
-            resizeGrid(0, newSizes, event);
-            resizeGrid(0, newSizes, event);
+          if (event.originalEvent.type !== 'resize')
```

Comment:
another nitpick can you move the opening curly brace on the same line as the `if`. Also I'm curious to know, what does this line help with? What other event type can show up here?

Reply:
oopsie! sorry! this was a try to break an endless loop, should not have been committed. removed it in the new push.